README.md

```diff
@@ -34,9 +34,9 @@ Additionally, it is recommended that one specifies Erlang/OTP, Elixir and `rebar
 using YAML strings, as these examples do, so that numbers like `23.0` don't
 end up being parsed as `23`, which is not equivalent.
 
-For pre-release Elixir versions, such as `1.11.0-rc.0`, use the full version
-specifier (`1.11.0-rc.0`). Pre-release versions are opt-in, so `1.11.x` will
-not match a pre-release.
+For pre-release versions, such as `v1.11.0-rc.0`, use the full version
+specifier (`v1.11.0-rc.0`) and set option `version-type` to `strict`. Pre-release versions are
```

Comment:
`version-type` will not always be required, for `rc`, but it's best to recommend it because if used results are guaranteed to be better.